apps/native/app/src/screens/inbox/inbox.tsx

```diff
@@ -536,6 +589,70 @@ export const InboxScreen: NavigationFunctionComponent<{
                     }
                   />
                 )}
+                {!!senderNationalId.length &&
+                  senderNationalId.map((senderId) => {
+                    const name = availableSenders.find((s) => s.id === senderId)
```

Comment:
This should be named sender

Reply:
senderNationalId is just an array of ids so I think senderId is appropriate there. If you are talking about s => s.id then I can change that to sender 😄